src/Steps/03_ClusterInformation.jsx

```diff
@@ -134,7 +134,7 @@ const ClusterInformation = ({ options, values, onChange }) => {
       <FormGroup label="Cluster Size" isRequired fieldId="cluster-size">
         <FormSelect
           aria-label="Cluster Size"
-          value={values.ocp_cluster_size}
+          value={values.ocp_cluster_size || ''}
```

Comment:
Just curious, does it cause a problem if these are null instead of empty string?

Reply:
Yes - I updated these across the board. Otherwise errors through in the browser console... The app still works - this just removes errors...